nestkernel/histentry.cpp

```diff
@@ -42,3 +41,11 @@ nest::histentry::histentry( double t,
   , access_counter_( access_counter )
 {
 }
+
+// member functions of histentry_cl
```

Comment:
this comment does not really help, either improve or remove

Reply:
Agreed, removed with 2e5d583.